pkg/scalers/aws_sqs_queue_scaler.go

```diff
@@ -89,39 +89,21 @@ func parseAwsSqsQueueMetadata(metadata, resolvedEnv map[string]string) (*awsSqsQ
 		return nil, fmt.Errorf("no queueURL given")
 	}
 
-	var keyName string
-	if keyName = metadata["awsAccessKeyID"]; keyName == "" {
```

Comment:
Let's hold off with removing this based on #453

Reply:
I fixed this and refactored the aws stuff a bit. I can apply it to the cloudwatch scaler also once this is merged.

Reply:
I think we're good now and this has been moved to `pkg/scalers/aws_iam_authorization.go`, right?

Reply:
yes